.github/workflows/tests.yaml

```diff
@@ -0,0 +1,46 @@
+on:
+  push:
+  pull_request:
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    strategy:
+      # all supported django version on lowest/highest supported python
+      # see https://www.djangoproject.com/download/#supported-versions
+      # and https://docs.djangoproject.com/en/4.2/faq/install/#what-python-version-can-i-use-with-django
+      matrix:
+        include:
+          - { dj: "4.1.*", py: "3.8" }
+          - { dj: "4.1.*", py: "3.11" }
+          - { dj: "4.2.*", py: "3.8" }
+          - { dj: "4.2.*", py: "3.12" }
+          - { dj: "5.0.*", py: "3.10" }
+          - { dj: "5.0.*", py: "3.12" }
```

Comment:
I assume that this is testing the oldest and latest versions of Python for each version of Django that current has upstream support?

Generally, I don't think that the oldest/latest approach is the best (I've seen code fail on individual versions of Python sandwiched between versions of Python under which the code passes), but since this will be using GHA from somebody else's account, I appreciate the attempt to minimize the amount that we use.

![I'll allow it](https://i.giphy.com/SmoCFhZCi1kzu.webp)